src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Microsoft.Azure.Batch.SoftwareEntitlement.Server.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <ApplicationIcon />
     <OutputTypeEx>library</OutputTypeEx>
     <StartupObject />
```

Comment:
Same thing with the `Debug|AnyCPU` `PropertyGroup` here. If you need `DebugType=full` for coverage, set it in the `Directory.Build.props` to make it less fiddly 👍

Reply:
Moved.